src/client/power-pages/actions-hub/CurrentSiteContext.ts

```diff
@@ -35,12 +40,16 @@ class CurrentSiteContext implements ICurrentSiteContext {
         return this._currentSiteId;
     }
 
+    public get currentSiteFolderPath(): string | null {
+        return this._currentSiteFolderPath;
```

Comment:
This gives path of the site folder or the folder containing it?

Reply:
The site folder.